index.html

```diff
@@ -90,7 +90,7 @@ <h2>A project for first-time contributions</h2>
 
         <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
         <div class="card">
-          <p class="name">Your name</p>
+          <p class="name">mugisha sheba</p>
```

Comment:
please do not modify the template
```suggestion
          <p class="name">Your name</p>
```